x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_connect/engine_api_integration.tsx

```diff
@@ -47,19 +47,18 @@ interface Tab {
 export const EngineApiIntegrationStage: React.FC = () => {
   const [selectedTab, setSelectedTab] = React.useState<TabId>('curl');
   const { engineName } = useValues(EngineViewLogic);
-  const enterpriseSearchUrl = getEnterpriseSearchUrl();
   const { apiKey } = useValues(EngineApiLogic);
 
   const Tabs: Record<TabId, Tab> = {
     curl: {
-      code: cURLSnippet(enterpriseSearchUrl, engineName, apiKey),
+      code: cURLSnippet(elasticsearchUrl(), engineName, apiKey),
       language: 'bash',
       title: i18n.translate('xpack.enterpriseSearch.content.engine.api.step3.curlTitle', {
         defaultMessage: 'cURL',
       }),
     },
     searchui: {
-      code: SearchUISnippet(enterpriseSearchUrl, engineName, apiKey),
+      code: SearchUISnippet(elasticsearchUrl(), engineName, apiKey),
```

Comment:
😬 The engines connector will need a rethink here too. Engine connector was designed for the ent-search engine and was going to evolve with it. Now that its _search, Elasticsearch connector here makes more sense.

No change needed here but we will need to pick up the conversation around what options we display here. I feel its a little premature to offer Search UI options and leaning to hide this, only show cURL for now. Il talk to Nick.